notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>Nice chart - although it is now slightly out of date - as I only require 1 data points within 1 minute - is it easy to re-create?</p><p> </p><p>This bottom comment "Run the code..." should be seperate the cell that put's the metrics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>I'll run through the whole pipeline again today and capture a new screenshot. </p><p> </p><p>Could you clarify what you mean about the bottom comment?</p><p> </p>

Reply:
<p>I'm suggesting that the following MD + Code cell should be added after the cell that outputs the metrics:</p><p> </p><p>Click through to the Alarm and CodePipeline with the links below.</p><p> </p><pre class="ql-syntax" spellcheck="false">metric_alarm_name = 'mlops-{}-metric-gt-threshold'.format(model_name)
HTML('''&lt;a target="_blank" href="https://{0}.console.aws.amazon.com/cloudwatch/home?region={0}#alarmsV2:alarm/{1}"&gt;CloudWatch Alarm&lt;/a&gt; triggers
&nbsp;&nbsp;&nbsp;&lt;a target="_blank" href="https://{0}.console.aws.amazon.com/codesuite/codepipeline/pipelines/{2}/executions?region={0}"&gt;Code Pipeline Execution&lt;/a&gt;'''.format(region, metric_alarm_name, pipeline_name))
</pre><p> </p><p>Once the pipeline is running again you can jump back up to [Wait for Training Job](#Wait-for-Training-Job)</p>